packages/core/frontend/src/router/index.ts

```diff
@@ -20,6 +17,11 @@ const routes: Array<RouteConfig> = [
     // which is lazy-loaded when the route is visited.
     component: () => import(/* webpackChunkName: "Workspace" */ "../views/Workspace.vue"),
     children: [
+      {
+        path: "/app/resources/hyperlink",
+        name: "Hyperlink",
+        component: () => import("../views/resources/Hyperlink.vue"),
```

Comment:
This should probably be `/app/resources/<id>`, and have it load the correct component once it fetches the resource with the given ID